xarray/core/treenode.py

```diff
@@ -456,14 +456,14 @@ def _get_item(self: Tree, path: str | NodePath) -> Tree | T_DataArray:
         for part in parts:
             if part == "..":
                 if current_node.parent is None:
-                    raise KeyError(f"Could not find node at {path}")
+                    raise KeyError(path)
```

Comment:
Is there a reason why you've made this error message less informative @shoyer ? The equivalent operation on a `Dataset` gives a more explicit message:
```python
xr.Dataset()['a']
```
```python
KeyError: "No variable named 'a'. Variables on the dataset include []"
```